_episodes/15-transferring-files.md

```diff
@@ -167,6 +167,14 @@ to transfer files.
 > files.
 {: .callout}
 
+> ## A note on rsync
+>
+> As you gain experience with transferring files, you may find the above methods limiting. The
+> [rsync](https://rsync.samba.org/) utility provides advanced features for file transfer and is
+> typically faster. It is especially useful for transferring large and/or many files and creating
+> synced backup folders.
```

Comment:
Please add examples to be on par with scp. I suggest to use the single file transfer example and the folder based example.

Reply:
To make myself concrete, please add one example of a download operation. After that, this PR is ready for merge from my point of view.

Reply:
Okay, I've added a download example. It should be merge-able, but let me know if I need to update the branch to the most recent commits.